.gitignore

```diff
@@ -27,3 +27,11 @@ server/
 account_local.screeps.com.js
 current-world/
 tmp-test-server-database/
+
+## ide
+.idea/
+**/*.iml
+
+## no ts
```

Comment:
nice but i would suggest, .gitignore changes to separate pr

Reply:
sonds resonable. will do.